### PR TITLE
Changed Host labels property to ListProp class.

### LINKED ISSUE #112: Host 'labels' property not right initialized
In Host class, we have this property : 
```
        'labels':
            StringProp(default=[], fill_brok=['full_status'], merging='join', split_on_coma=True),
```

It's a string and default can't be [], or default is right [] but type is list, not string

### PR DESCRIPTION
**Pull request by gst**
Should fix #112

### CODE REVIEW

**Comment by Seb-Solon** [comment]
Need rebase + test. 
FOr test you can have a look at https://github.com/Alignak-monitoring/alignak/blob/develop/test/test_end_parsing_types.py (iterate on host and check type. It will be a bit hard)

**Comment by gst** [comment]
there it is. 

up to your merge :p

**Comment by gst** [comment]
ping

File: test/test_end_parsing_types.py [hunk 1]
```diff
@@ -148,46 +175,16 @@ def test_types(self):
...
-                        print("Skipping %s " % prop)
+        ###############
+
+        self.check_objects_from(self.conf.arbiters)
+
+        self.check_objects_from(self.conf.contacts)
+
+        self.check_objects_from(self.conf.notificationways)
+
+        self.check_objects_from(self.conf.hosts)
```
Comment (Seb-Solon) [inline test/test_end_parsing_types.py]: What about : 
```
for objs in [self.conf.arbiters, self.conf.contacts, self.conf.notificationways, self.conf.hosts]:
    self.check_objects_from(objs)
```

**Comment by Seb-Solon** [comment]
Can you also reword f814b5a ?

**Comment by gst** [comment]
> Can you also reword f814b5a ?

heuh.. why ? the message corresponds well to the commit imo..

**Comment by gst** [comment]
ready to merge (once travis done..)

**Comment by gst** [comment]
ready to merge (once travis done..)

**Comment by Seb-Solon** [comment]
"Simpler is better" looks like "The sky is blue" for me actually. It doesn't help me a lot to get what is done in the commit. Same thing for "Dry". 

Sorry if I have diffuculties to understand your commit, maybe somebody else has no problem with it and can review those PR quicker than me.

**Comment by gst** [comment]
Simpler commit message expanded (the commit content was quite self-explanable I found, but ok).
For the DRY : hey it's you who asked me to change that ;)

**Comment by Seb-Solon** [comment]
Yup I suggest an enhancement, not a single word commit. Basically a squash / fixup was fine. I wanted to do it myself but since it's on your repo I can't just force push on that PR and wait for Travis. 

I did the rebase, tell me if you want me to push and make an new PR. We spent a lot of time in my opinion, I'll be happy to merge that today

**Comment by gst** [comment]
squashed that.
I usually prefer (and it's usually preferable/better) many simple commits than a (or few) bigger one(s).

**Pull request merged**